### PR TITLE
BD-3164, BD-3165 - AI Liquid Assistant overview and prompt guide

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-3164** & **BD-3165**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Including context in your prompt helps the assistant tailor its responses to better suit your needs. You can also include details from your campaign, message brief, or brainstorming document to bring the assistant up to speed.
+
+### Be specific
+
+The AI Liquid Assistant can ask follow-up questions, but providing details upfront can lead to more precise results sooner. Consider including details such as:
+
+- Any known preferences or requirements for the message
+- Instructions on how to handle situations, such as a lack of responses from the message recipient or fallback message options
+- When asking for Liquid that uses Connected Content, documentation for the API endpoint or a sample API response or both
```
Comment (lydia-xie) [inline _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md:43]:
```suggestion

- Any known preferences or requirements for the message
- Instructions on how to handle situations, such as a lack of responses from the message recipient or fallback message options
- When asking for Liquid that uses Connected Content, documentation for the API endpoint, a sample API response, or both
```

File: _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+The AI Liquid Assistant is trained to understand natural language. Chat with it as you would with a coworker when asking for help. This makes it easier for the assistant to comprehend your needs and provide accurate assistance.
+
+### Give context
+
+Providing context helps the AI Liquid Assistant understand the bigger picture surrounding your project. It's helpful to include context such as:
+
+- Your company name and industry
+- A compaign you're working on, such as Black Friday or holiday sales
+- Your goal, ushc as increasing your click-through rate
+- Specific custom attributes you want to include in your message
```
Comment (lydia-xie) [inline _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md:33]:
```suggestion
Providing context helps the AI Liquid Assistant understand the bigger picture surrounding your project. It's helpful to include context such as:

- Your company name and industry
- A campaign you're working on, such as Black Friday or holiday sales
- Your goal, such as increasing your click-through rate
- Specific custom attributes you want to include in your message
```

File: _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+noindex: true
+---
+
+# About prompting the AI Liquid Assistant
+
+> Prompting is the process of giving the AI Liquid Assistant clear instructions on what you want it to do.
+
+The AI Liquid Assistant is designed to function as a best-in-class marketer that's an expert at using Braze and Liquid to create impactful marketing messages, but it still requires guidance on your particular use cases or questions.
+
+There is no single correct way to prompt the assistant, and you can always add further deatils to your prompts during your conversation. This means you don't need to spend a lot of time crafting the "perfect" initial prompt. However, you can get the most out of the assistant by following some best practices.
```
Comment (lydia-xie) [inline _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md:18]:
```suggestion
The AI Liquid Assistant is designed to function as a best-in-class marketer who's an expert at using Braze and Liquid to create impactful marketing messages, but it still requires guidance on your particular use cases or questions.

There is no single correct way to prompt the assistant, and you can always add further details to your prompts during your conversation. This means you don't need to spend a lot of time crafting the "perfect" initial prompt. However, you can get the most out of the assistant by following some best practices.
```

File: _docs/_user_guide/sage_ai/generative_ai/ai_liquid.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+The AI Liquid Assistant is designed to help you write effective Liquid code tailored to your marketing needs. Trained on both Liquid syntax and how marketers utilize Liquid in their messages, our AI understands the nuances of crafting personalized content. Additionally, by providing the AI Liquid Assistant your custom attribute names (such as `favourite_color`) and data types (such as boolean and string), our AI Assistant ensures your messages are precisely targeted and aligned with your goals.
+
+## Generating Liquid code
+
+Access the AI Liquid Assistant by selecting the [insert Braze icon] icon above the **Message** composer. You can select from the prompts listed in the assistant or enter your own prompt into the textbox.  
+
+After you provide the assistant with information about the message you want to create, it will generate Liquid code that you can view by selecting **Update composer**. You can select **Regenerate** to generate another message for the same prompt or **Undo update** to remove the generated message from the composer and revert the prior message contents.
+
+### Example assistant prompts
```
Comment (lydia-xie) [inline _docs/_user_guide/sage_ai/generative_ai/ai_liquid.md:29]:
```suggestion
### Example assistant prompts
```
Thoughts on linking to the About Prompting the AI Liquid Assistant doc here? 🤔

File: _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md (hunk 3 above)
Comment (lydia-xie) [reply _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md:18]: Do we need to add an important alert here too to flag that this is in beta? 🤔

**Review by lydia-xie — APPROVED** [review]
lgtm! left a handful of suggestions 🫡

File: _docs/_user_guide/sage_ai/generative_ai/ai_liquid.md (hunk 4 above)
Comment (rachel-feinberg) [reply _docs/_user_guide/sage_ai/generative_ai/ai_liquid.md:29]: That doc is hidden, so I'm leaving it out for now!

File: _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md (hunk 3 above)
Comment (rachel-feinberg) [reply _docs/_user_guide/sage_ai/generative_ai/ai_liquid/ai_liquid_prompts.md:18]: Good catch! I'll add that in

**Pull request merged**